blocks/bricks/conv.py

```diff
@@ -185,7 +185,7 @@ class ConvolutionalActivation(Sequence, Initializable):
         The application method to apply after convolution (i.e.
         the nonlinear activation function)
 
-    See :class:`Convolutional` for explanation of other parameters.
+    See: :class:`Convolutional`
```

Comment:
Not sure if this is much of an improvement, considering it's now a bit unclear why you should look at `Convolutional`. The proper NumPy docstring syntax by the way is:
```
See Also
--------
:class:`Convolutional`
```